### PR TITLE
Improve oboe-errors challenge

### LINKED ISSUE #13455: Cannot read property 'length' of null
Challenge [catch-off-by-one-errors-when-using-indexing](http://beta.freecodecamp.com/en/challenges/debugging/catch-off-by-one-errors-when-using-indexing) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Its throwing an error: "Cannot read property 'length' of null" however; in my browser this works as expected.

```js


function countToFive() {
  let firstFive = "12345";
  
  for (let i = 0; i < firstFive.length; i++) {
    console.log(firstFive[i]);
  }
  
  return firstFive;
}

countToFive();

```

### PR DESCRIPTION
**Pull request by Manish-Giri**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13455 

#### Description

Added comments to indicate the line of code to be modified in `challengeSeed`. Also amended a few of the test messages to make it more descriptive.

Closes #13455.

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json [hunk 1]
```diff
@@ -294,7 +294,9 @@
         "function countToFive() {",
         "  let firstFive = \"12345\";",
         "  let len = firstFive.length;",
+        "  //Fix this line",
```
Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: Nitpicking, but can you put a space in front of `Fix`? Most of the comments I've seen in the seed have a space between the `//` and the comments themselves.

File: seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json [hunk 2]
```diff
@@ -294,7 +294,9 @@
...
         "  for (let i = 1; i <= len; i++) {",
+        "  //Do not alter code below this line",
```
Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: Same here as well.

File: seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json (hunk 1 above)
Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: Maybe even be more specific with `// Fix the line below`

**Review by erictleung — COMMENTED** [review]
Just some small changes I think will make it consistent with the rest of the challenges.

File: seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json (hunk 1 above)
Comment (Manish-Giri) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: @erictleung Will do! Thanks for looking! Are the test messages okay?

Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: @Manish-Giri looks good to me!

Comment (Manish-Giri) [inline seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json]: @erictleung Thx! Just made the changes..

**Review by JosephLivengood — APPROVED** [review]
With that edit LGTM 👍 !

**Review by erictleung — APPROVED** [review]
LGMT! 👍

**Pull request merged**